test/gtest/stringify.cpp

```diff
@@ -254,7 +254,7 @@ std::vector<SuffixTree::RepeatedSubstring>
 repeatSubstrings(std::vector<uint32_t> hashString) {
   SuffixTree st(hashString);
   std::vector<SuffixTree::RepeatedSubstring> substrings =
```

Comment:
```suggestion
  auto substrings =
```
(now that we write out the full type below anyhow, I think it makes sense to avoid repeating it above)

Reply:
went with the below option from Thomas, but thank you!